examples/differential_drive/Differential Drive.ipynb

```diff
@@ -1249,7 +1249,7 @@
     "x0 = [0,] * 15                         # Start out at origin, unmoving\n",
     "xs = odeint(right_hand_side, x0, ts, args=(ts, us, values))\n",
     "\n",
-    "plt.plot(xs[:,5], xs[:,6])\n",
+    "plt.plot(xs[:,0], xs[:,1])\n",
```

Comment:
So was this plotting the velocities?

Reply:
Yes it was, so not corresponding to the plot